meetings/2021-09-01.md

```diff
@@ -0,0 +1,107 @@
+of operations and WAC currently defines mapping of HTTP methods to access modes only to inform developers. 
+Currently, operations are normative whereas specific HTTP methods are non-normative. 
+The reason is eg. PUT typically needs Write mode, but PUT + If-None-Match: * would need Write and Read.
+
+**Henry** 
+  To Eric's point above: the reason I am reaching out to Category Theory, is that 
+it is the mathematics of structures. What it tells us is that Dynamic Systems can be
+modelled using essentially just these Read and Write operations. See the maths
+discussed on [web-cats](https://gitlab.com/web-cats/CG/-/issues/28#note_666179055). 
+Since we can also do [without Control mode](https://github.com/solid/web-access-control-spec/issues/94), 
+that indicates that it is not a structural necessary piece at the level of Read and Write.
+It should be modelled differently (via the `Link` header).
+  
+**Sarven** PATCH can include DELETE, so in that case it will require read and write.
+
+**Pavlik** can we have Write without Read?
+
+**Pavlik** How do we see that List mode, can we separate reading container description and the list of containment triples?
+
+**Sarven** List would be limited type of Read. It seems that we are coming to consensus on listing containers. We have this understanding of what container is and what container description can be. The starting point seems to be that description includes containment. There is legitimate case to get some useful information.
+
+**Justin** having Update / Modify on container means for me that one can change the non-server managed (containement) triples. To change those one would need Created (which would result in adding) and Delete (removing)
+
+**Sarven** I'd like to see actual applications before hypothesising.
+
+**Henry** I think we should take some use cases, try to implement them as we have done in in the evaluation
+section https://github.com/solid/authorization-panel/tree/main/proposals/evaluation
+and see if we can use the minimal Read/Write modes, and then we can try our best using existing modes, and then see why exactly a new mode would be necessary.
+
+**Pavlik** I don't think Append on a container should imply Create, it should only allow adding new statements to the container resource.
+
+**Justin** Access Modes should provide specificity to define access exactly the way one wants.
+When it comes to dogfooding it hard if current systems don't support it. We do have use cases which detail Create, Delete, Update and real world cases when current access modes lead to over permissioning. I would like to understand what else is requested besides use cases that are already included.
+
+**Henry** re append, I thought a bit about [it here](https**//gitlab.com/web-cats/CG/-/issues/28#note_666387691) with regard to lenses
+
+**Henry** I think use cases are very good. I think we should do what we did in evaluation section, seeing how it would work with HTTP methods etc.
+
+**Sarven** GOTO 57
```

Comment:
@csarven what does that mean? Something may have gotten lost in the notes there.